packages/adapters/solana/src/client.ts

```diff
@@ -129,11 +129,20 @@ export class SolanaAdapter implements ChainAdapter {
 
     this.appKit = appKit
     this.options = options
-    this.caipNetworks = options.networks
-    this.defaultCaipNetwork = SolHelpersUtil.getChainFromCaip(
-      options.networks,
-      SafeLocalStorage.getItem(SafeLocalStorageKeys.ACTIVE_CAIP_NETWORK_ID)
+    this.caipNetworks = options.networks.map(network =>
+      SolHelpersUtil.isSolanaChainNameSpace(network)
+        ? SolHelpersUtil.getChainFromCaip({
```

Comment:
Same for this one, do we still need to use `getChainFromCaip` function? I think extending them on AppKit client should be enough?

Reply:
Yes i even removed `SolHelpersUtil.getChainFromCaip` which is not needed because we can get the caip networks from AppKit client directly